js/views/modals/listingDetail/Listing.js

```diff
@@ -127,6 +202,20 @@ export default class extends BaseModal {
     }
   }
 
+  setShippingDestination(e) {
```

Comment:
Kinda minor, but I would name this `onSetShippingDestination`. It expects a jQuery event object, so it's really only designed to be used as an event handler. Without the 'on' part, it seems like an API method of the view to set a shipping destination.